Description: enforce proxies better
 This just ensures that all available http_proxy environment variables are set
 .
 apt-transport-i2phttp (0.0~git20190224.6dc102d-1) UNRELEASED; urgency=medium
 .
   [ idk ]
   * Initial release (Closes: TODO)
 .
   [ idk ]
   * set the http_proxy(lowercase) environment variable too
Author: idk <hankhill19580@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: upstream
Last-Update: 2019-02-28

--- apt-transport-i2phttp-0.0~git20190224.6dc102d.orig/main.go
+++ apt-transport-i2phttp-0.0~git20190224.6dc102d/main.go
@@ -58,6 +58,7 @@ func ReadInConfig() {
 	if _, err := os.Stat(os.Getenv("APT_TRANSPORT_I2PHTTP_CONF")); os.IsNotExist(err) {
 		os.Setenv("I2P_HTTP_PROXY", "http://127.0.0.1:4444")
 		os.Setenv("HTTP_PROXY", "http://127.0.0.1:4444")
+        os.Setenv("http_proxy", "http://127.0.0.1:4444")
 	} else if err != nil {
 		log.Fatal(err)
 	} else {
@@ -80,12 +81,15 @@ func ReadInConfig() {
 		}
 		os.Setenv("I2P_HTTP_PROXY", "http://"+host+":"+port)
 		os.Setenv("HTTP_PROXY", "http://"+host+":"+port)
+        os.Setenv("http_proxy", "http://"+host+":"+port)
 	}
 	if os.Getenv("I2P_HTTP_PROXY") == "" {
 		os.Setenv("I2P_HTTP_PROXY", "http://127.0.0.1:4444")
 		os.Setenv("HTTP_PROXY", "http://127.0.0.1:4444")
+        os.Setenv("http_proxy", "http://127.0.0.1:4444")
 	} else {
 		os.Setenv("HTTP_PROXY", os.Getenv("I2P_HTTP_PROXY"))
+        os.Setenv("http_proxy", os.Getenv("I2P_HTTP_PROXY"))
 	}
 }
 
